org-formation/600-access/_tasks.yaml

```diff
@@ -115,6 +115,51 @@ CnbCIServiceAccount:
       - !Ref CnbAccount
     Region: !Ref primaryRegion
 
+# Service account for GHA portal deployer in SageIT account
+PortalDeployGhaServiceAccount:
+  Type: update-stacks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.3.8/templates/IAM/service-account.yaml
+  StackName: portal-deployer-gha-service-account
+  Parameters:
+    PolicyDocument: |
+      {
+        "Version": "2012-10-17",
+        "Statement": [
+          {
+            "Sid": "PublishToS3Web",
+            "Action": [
+              "acm:ListCertificates",
+              "cloudfront:CreateDistribution",
+              "cloudfront:DeleteDistribution",
+              "cloudfront:GetDistribution",
+              "cloudfront:GetDistributionConfig",
+              "cloudfront:ListDistributions",
+              "cloudfront:UpdateDistribution",
+              "cloudfront:ListCloudFrontOriginAccessIdentities",
+              "cloudfront:CreateInvalidation",
+              "cloudfront:GetInvalidation",
+              "cloudfront:ListInvalidations",
+              "cloudfront:TagResource",
+              "s3:PutBucketWebsite",
+              "s3:PutObject",
+              "s3:PutObjectAcl",
+              "s3:GetObject",
+              "s3:ListBucket",
+              "s3:DeleteObject",
+              "s3:GetBucketLocation"
+            ],
+            "Effect": "Allow",
+            "Resource": "*"
```

Comment:
Should this be further scoped to only allow access to specific buckets?

Reply:
Sure, but I'd say 70% of buckets (maybe more if we clean up) and 80% of distributions are in the list...
Should I try to deny on stuff we know is critical (e.g. essentials*)?

Reply:
It probably depends on the use case.  i took a look at the referenced issue PLFM-1695 and didn't see anything that explains what this role is for.